src/games/ColorGame.js

```diff
@@ -0,0 +1,499 @@
+				{ answerText: 'Yellow', isCorrect: false},
+				{ answerText: 'Red', isCorrect: true},
+				{ answerText: 'Blue', isCorrect: false},
+			],
+		},
+		{
+			answer: 'Green',
+			questionText: 'Green',
+			answerOptions: [
+				{ answerText: 'Green', isCorrect: true },
+				{ answerText: 'Yellow', isCorrect: false },
+				{ answerText: 'Purple', isCorrect: false },
+				{ answerText: 'Red', isCorrect: false },
+			],
+		},
+		{
+			answer: 'Purple',
+			questionText: 'Green',
+			answerOptions: [
+				{ answerText: 'Orange', isCorrect: false },
+				{ answerText: 'Purple', isCorrect: true },
+				{ answerText: 'Yellow', isCorrect: false },
+				{ answerText: 'Green', isCorrect: false },
+			],
+		},
+
+
+
+		//Purple Questions
+		{
+			answer: 'Blue',
+			questionText: 'Purple',
+			answerOptions: [
+				{ answerText: 'Blue ', isCorrect: true },
+				{ answerText: 'Yellow', isCorrect: false },
+				{ answerText: 'Purple', isCorrect: false },
+				{ answerText: 'Orange', isCorrect: false },
+			],
+		},
+		{
+			answer: 'Yellow',
+			questionText: 'Purple',
+			answerOptions: [
+				{ answerText: 'Blue ', isCorrect: false },
+				{ answerText: 'Yellow', isCorrect: true },
+				{ answerText: 'Purple', isCorrect: false },
+				{ answerText: 'Orange', isCorrect: false },
+			],
+		},
+		{
+			answer: 'Orange',
+			questionText: 'Purple',
+			answerOptions: [
+				{ answerText: 'Purple', isCorrect: false },
+				{ answerText: 'Yellow', isCorrect: false },
+				{ answerText: 'Red', isCorrect: false },
+				{ answerText: 'Orange', isCorrect: true },
+			],
+		},
+		{
+			answer: 'Red',
+			questionText: 'Purple',
+			answerOptions: [
+				{ answerText: 'Purple', isCorrect: false},
+				{ answerText: 'Yellow', isCorrect: false},
+				{ answerText: 'Red', isCorrect: true},
+				{ answerText: 'Blue', isCorrect: false},
+			],
+		},
+		{
+			answer: 'Green',
+			questionText: 'Purple',
+			answerOptions: [
+				{ answerText: 'Green', isCorrect: true },
+				{ answerText: 'Yellow', isCorrect: false },
+				{ answerText: 'Purple', isCorrect: false },
+				{ answerText: 'Blue', isCorrect: false },
+			],
+		},
+		{
+			answer: 'Purple',
+			questionText: 'Purple',
+			answerOptions: [
+				{ answerText: 'Orange', isCorrect: false },
+				{ answerText: 'Purple', isCorrect: true },
+				{ answerText:  'Yellow', isCorrect: false },
+				{ answerText: 'Green', isCorrect: false },
+			],
+		},
+			
+	];
+
+	const max = 30;
+
+	const [gameState, setGameState] = useState(GameState.InProgress);
+	const [currentQuestion, setCurrentQuestion] = useState(0);
+	const [showScore, setShowScore] = useState(false);
+	const [score, setScore] = useState(0);
+	const [initialMinute,initialSeconds] = [0,max];
+    const [minutes, setMinutes ] = useState(initialMinute);
+    const [seconds, setSeconds ] =  useState(initialSeconds);
+	const [start, setStart] = useState(0);
+	const [count, setCount] = useState(seconds);
+	var timer;
+	
+	const GAME_TYPE = 4;
```

Comment:
prefer to pull this from here: https://github.com/san0014s/Cog-Web-App/blob/main/src/constants/GamesConstants.js#L1

Reply:
It is. I just had that there from earlier. I just removed the GAME_TYPE const from the code. I can push it here in a second